.ci/beats-webhook.groovy

```diff
@@ -51,7 +51,7 @@ pipeline {
      printPostContent: false,
      silentResponse: true,
      regexpFilterText: '$ref-$x_github_event',
-     regexpFilterExpression: '^(refs/tags/current|refs/heads/master/.+)-comment$'
+     regexpFilterExpression: '^(refs/tags/current|refs/heads/master/.+|refs/heads/main/.+)-comment$'
```

Comment:
Are we ready to change this?

Reply:
It's not replacing master but adding it. The reason was to avoid forgetting this change in the future, so I decided to add it to the regex then it should be harmless